far/FarUkr.hlf.m4

```diff
@@ -5110,17 +5108,21 @@ zero-size expression.
  #{,m}#   - ^<wrap>equals to {0,m}
  #{,m}?#  - ^<wrap>equals to {0,m}?
 
- #"Greedy" and "not greedy" quantifiers#
+ #"Жадібні" та "не жадібний" квантифікатори#
+
```

Comment:
Please avoid adding or removing empty lines in HLFs.
Differences in paragraphs structure [won't pass](https://github.com/FarGroup/FarManager/actions/runs/5847752036/job/15854320652?pr=720) the build check.

Reply:
fixed this paragraph